.github/workflows/ci.yml

```diff
@@ -549,10 +549,12 @@ jobs:
         with:
           toolchain: ${{ env.RUST_VERSION }}
       - uses: Swatinem/rust-cache@v2
+      - name: Setup cargo-nextest
```

Comment:
You might want to look at moonrepo/setup-rust, which uses cargo-binstall under the hood: https://github.com/FuelLabs/sway/blob/56997219cbfca3745d5402a9d76ca6efc6421d01/.github/workflows/benchmark.yml#L32

Reply:
Thanks @sdankel . Added in [a7d9fa0](https://github.com/FuelLabs/sway/pull/6521/commits/a7d9fa0f78816726fb03e80fbac006d1b001c6d7)